app/features/dynamo/tools.py

```diff
@@ -28,7 +28,7 @@ def read_text_file(file_path):
         return file.read()
 
 # Summarize chain
-def summarize_transcript(youtube_url: str, max_video_length=600, verbose=False) -> str:
```

Comment:
max_video_length is a necessary part of the application. Please include it in the parameters as before.